frontend/src/charts/TableChart.tsx

```diff
@@ -52,6 +52,28 @@ export function TableChart(props: TableChartProps) {
     },
   ].concat(columns);
 
+  if (breakdownVar === "age") {
```

Comment:
Can you make this logic simpler if you assume that anything that is unbounded (i.e. "x+") will be at the end of the list?